src/tools/unidasm.cpp

```diff
@@ -637,10 +637,13 @@ static const dasm_table_entry dasm_table[] =
 	{ "superfx",         le,  0, []() -> util::disasm_interface * { return new superfx_disassembler(&superfx_unidasm); } },
 	{ "t11",             le,  0, []() -> util::disasm_interface * { return new t11_disassembler; } },
 	{ "tlcs870",         le,  0, []() -> util::disasm_interface * { return new tlcs870_disassembler; } },
-	{ "tlcs900",         le,  0, []() -> util::disasm_interface * { return new tlcs900_disassembler; } },
 	{ "tmp90c051",       le,  0, []() -> util::disasm_interface * { return new tmp90c051_disassembler; } },
 	{ "tmp90840",        le,  0, []() -> util::disasm_interface * { return new tmp90840_disassembler; } },
 	{ "tmp90844",        le,  0, []() -> util::disasm_interface * { return new tmp90844_disassembler; } },
+	{ "tmp94c241",       le,  0, []() -> util::disasm_interface * { return new tmp94c241_disassembler; } },
+	{ "tmp95c061",       le,  0, []() -> util::disasm_interface * { return new tmp95c061_disassembler; } },
+	{ "tmp95c063",       le,  0, []() -> util::disasm_interface * { return new tmp95c063_disassembler; } },
+	{ "tmp96c141",       le,  0, []() -> util::disasm_interface * { return new tmp96c141_disassembler; } },
```

Comment:
I think there’s still potential value in having a generic TLCS900 core disassembler without specifying a specific device for the purpose of investigating code when the exact device it targets is unknown or unsupported.